packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -267,9 +267,16 @@ class PointerEventConverter {
                 position: position,
                 embedderId: datum.embedderId,
               );
+            case ui.PointerSignalKind.scale:
```

Comment:
Oh yeah it looks like there is a bad merge in here.

Reply:
Sorry, fixed.